libs/ui/src/pure/MenuBar/index.tsx

```diff
@@ -351,15 +349,6 @@ export const MenuBar = ({ navItems, theme, productVariant, additionalContent }:
 
   const isMobile = useMediaQuery(upToLarge)
 
-  // Construct the base and mobile product variants
-  const baseProductVariant = `${productVariant}${theme.charAt(0).toUpperCase()}${theme.slice(1)}Mode`
-  const iconOnlyVariant = `${productVariant}IconOnly${theme.charAt(0).toUpperCase()}${theme.slice(1)}Mode`
-
-  // Determine the appropriate logo variant
-  const logoVariant = isMobile && LOGO_MAP[iconOnlyVariant] ? iconOnlyVariant : baseProductVariant
-
-  console.log('Using logo variant:', logoVariant) // Log to verify correct variant
```

Comment:
this is very fragile. Replacing this with strong types, so if you add a product, you will get a compile error if you don't add the styles for the logo